gradle/versions.gradle

```diff
@@ -0,0 +1,39 @@
+// Variables for entire project
+ext {
+  // Setup
+  minSdkVersion = 14
+  targetSdkVersion = 25
+  compileSdkVersion = 25
+  buildToolsVersion = "25.0.0"
+
+  // Common versions
+  supportLibraryVersion = "25.0.0"
+}
+
+// Plugins
+ext.plugs = [
+    "gradle"                  : "com.android.tools.build:gradle:2.2.3",
+    "androidMavenGradlePlugin": "com.github.dcendents:android-maven-gradle-plugin:1.4.1"
+]
+
+// Dependencies
+ext.deps = [
```

Comment:
@weijjia This project has multi `build.gradle` files. Aggregating all of the GAVs(groupid:artifactid:version) together, we can better keep track of the versions we are using.